_layouts/default.html

```diff
@@ -0,0 +1,171 @@
+  <script type="text/javascript">
+    $j(function(){
+        $j('.slide-out-div').tabSlideOut({
+            tabHandle: '.handle',                     //class of the element that will become your tab
+            pathToTabImage: '/images/change.png', //path to the image for the tab //Optionally can be set using css
+            imageHeight: '148px',                     //height of tab image           //Optionally can be set using css
+            imageWidth: '37px',                       //width of tab image            //Optionally can be set using css
+            tabLocation: 'left',                      //side of screen where tab lives, top, right, bottom, or left
+            speed: 300,                               //speed of animation
+            action: 'click',                          //options: 'click' or 'hover', action to trigger animation
+            topPos: '87px',                          //position from the top/ use if tabLocation is left or right
+            leftPos: '20px',                          //position from left/ use if tabLocation is bottom or top
+            fixedPosition: true                      //options: true makes it stick(fixed position) on scroll
+        });
+    
+    });
+  </script>
+  <script type="text/javascript">
+    var $j = jQuery.noConflict();
+    $j(document).ready(function() {
+        $j(".list-3 li:even").addClass("even");
+    });
+  </script>
+  <script type="text/javascript">
+    var $j = jQuery.noConflict();
+    $j(document).ready(function() {
+        $j(".list-6 li:even").addClass("even");
+    });
+  </script>
+  <script type="text/javascript">
+    var $j = jQuery.noConflict();
+    $j(document).ready(function() {
+        $j('input[type="text"]').addClass("idleField");
+                $j('input[type="text"]').focus(function() {
+                 $j(this).removeClass("idleField").addClass("focusField");
+                 if (this.value == this.defaultValue){ 
+                  this.value = '';
+         }
+         if(this.value != this.defaultValue){
+              this.select();
+             }
+            });
+            $j('input[type="text"]').blur(function() {
+             $j(this).removeClass("focusField").addClass("idleField");
+                 if ($j.trim(this.value) == ''){
+              this.value = (this.defaultValue ? this.defaultValue : '');
+         }
+            });
+    });
+  </script>
+  <!-- WP-Cufon Plugin 1.6.10 START  -->
+  <script type="text/javascript" src="{{site.baseurl}}/js/cufon-yui.js"></script>
+  <!-- WP-Cufon Fonts found  -->
+  <script src="{{site.baseurl}}/js/AvantGarde_Bk_BT_400.font.js" type="text/javascript"></script>
+  <!-- WP-Cufon Plugin Replacements --> 
+  <script type="text/javascript">
+    Cufon.replace('h3, #faded ul li strong, .entry-title, .support-posts ul li i, .list-1 dt, .feature-posts ul li i, .breadcrumbs', { fontFamily: 'AvantGarde Bk BT', hover: 'true' }); 
+  </script>            
```

Comment:
I removed this, it was very easy to take out.  You will notice that the fonts on my example site now look different than your current site.  I'm not sure if that matters to you or not?  It is a pretty subtle change if you ask me, but certainly wanted to make sure you were aware of that change.  If you don't like the font change, then I can revert this change and put that plugin back into the site.  I think a css refactor would be needed to try and remove the plugin and keep the fonts the same?